resources/assets/js/models/types.ts

```diff
@@ -98,7 +98,10 @@ export interface AwardRecognitionType {
 }
 
 export interface Classification {
+  id: number;
   key: string;
+  defaultMessage: string;
+  description: string;
```

Comment:
After implementing the migration changes, these will need to be typed as `localizedField[Non]Null` to make use of the expected json structure